[DEBUG] 2019-09-17 16:55:47,418 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 16:56:47,619 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 16:57:47,801 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 16:58:47,981 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 16:59:48,150 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:00:48,367 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:01:48,543 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:02:48,754 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:03:48,937 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:04:49,124 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:05:49,316 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:06:49,494 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:07:49,679 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:08:49,881 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:09:50,068 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:10:50,239 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:11:50,425 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:12:50,590 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:13:50,764 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:14:50,945 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:15:51,117 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:16:51,316 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:17:51,503 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:18:51,677 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:19:51,857 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:20:52,062 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:21:52,252 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:22:52,442 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:23:52,638 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:24:52,832 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:25:53,012 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:26:53,197 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:27:53,397 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:28:53,568 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:29:53,755 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:30:53,922 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:31:54,101 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:32:54,295 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:33:54,478 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:34:54,663 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:35:54,847 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:36:55,035 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:37:55,226 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:38:55,406 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:39:55,592 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:40:55,779 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:41:55,962 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:42:56,158 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:43:56,358 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:44:56,554 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:45:56,742 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:46:56,920 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:47:57,102 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:48:57,306 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-17 17:49:57,486 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
                                                                                                           [DEBUG] 2019-09-27 15:47:33,596 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:48:33,757 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:49:33,900 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:50:34,060 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:51:34,185 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:52:34,312 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:53:34,476 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:54:34,647 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:55:34,822 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:56:35,019 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:57:35,207 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:58:35,391 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 15:59:35,564 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:00:35,802 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:01:35,960 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:02:36,148 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:03:36,316 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:04:36,513 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:05:36,675 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:06:36,852 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:07:37,042 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:08:37,228 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:09:37,402 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:10:37,561 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:11:37,729 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:12:37,923 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:13:38,126 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:14:38,307 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:15:38,482 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:16:38,659 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:17:38,839 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:18:39,023 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:19:39,197 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:20:39,380 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:21:39,542 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:22:39,728 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:23:39,909 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:24:40,077 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:25:40,271 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:26:40,447 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:27:40,635 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:28:40,823 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:29:41,002 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:30:41,186 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:31:41,374 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:32:41,568 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:33:41,743 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:34:41,923 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:35:42,100 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:36:42,297 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:37:42,474 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:38:42,651 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:39:42,833 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:40:43,015 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:41:43,201 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:42:43,381 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:43:43,569 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:44:43,757 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:45:43,939 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:46:44,122 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:47:44,295 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:48:44,483 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:49:44,636 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:50:44,803 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:51:44,988 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:52:45,177 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:53:45,354 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:54:45,542 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:55:45,714 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:56:45,897 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:57:46,071 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:58:46,247 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 16:59:46,430 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:00:46,651 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:01:46,828 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:02:47,022 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:03:47,194 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:04:47,384 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:05:47,559 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:06:47,712 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:07:47,894 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:08:48,082 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:09:48,250 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:10:48,417 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:11:48,590 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:12:48,770 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:13:48,960 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:14:49,133 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:15:49,319 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:16:49,515 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:17:49,702 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:18:49,875 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:19:50,047 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:20:50,229 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:21:50,404 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:22:50,572 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:23:50,768 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:24:50,943 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:25:51,122 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:26:51,297 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:27:51,481 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:28:51,676 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:29:51,846 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:30:52,028 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:31:52,207 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:32:52,388 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:33:52,567 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:34:52,743 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:35:52,929 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:36:53,117 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:37:53,293 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:38:53,477 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:39:53,663 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:40:53,859 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:41:54,039 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:42:54,205 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:43:54,385 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:44:54,562 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:45:54,737 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:46:54,914 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:47:55,086 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:48:55,277 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:49:55,463 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:50:55,628 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:51:55,806 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:52:55,994 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:53:56,163 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[DEBUG] 2019-09-27 17:54:56,348 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Queued events for site moms-site2
[ERROR] 2019-09-28 09:16:43,085 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:16:43,172 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:17:43,318 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:17:43,320 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:18:43,462 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:18:43,464 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:19:43,594 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:19:43,596 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:20:43,747 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:20:43,749 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:21:43,903 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:21:43,905 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:22:44,068 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:22:44,070 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:23:44,242 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:23:44,243 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:24:44,406 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:24:44,408 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:25:44,584 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:25:44,586 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:26:44,761 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:26:44,763 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:27:44,936 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:27:44,939 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:28:45,108 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:28:45,111 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:29:45,281 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:29:45,283 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:30:45,442 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:30:45,444 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:31:45,614 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:31:45,616 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:32:45,774 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:32:45,776 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:33:45,941 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:33:45,943 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:34:46,114 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:34:46,116 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:35:46,295 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:35:46,297 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:36:46,469 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:36:46,471 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:37:46,646 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:37:46,648 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:38:46,787 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:38:46,789 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:39:46,929 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site1
Exception in Enqueue Events for Site moms-site1
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

[ERROR] 2019-09-28 09:39:46,931 | /home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py:
Site: moms-site2
Exception in Enqueue Events for Site moms-site2
Traceback (most recent call last):
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 93, in enqueue_events_for_site
    if is_scheduler_disabled():
  File "/home/umoms/moms-bench/apps/frappe/frappe/utils/scheduler.py", line 237, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 642, in get_single_value
    tabSingles where doctype=%s and field=%s""", (doctype, fieldname))
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 165, in sql
    self.connect()
  File "/home/umoms/moms-bench/apps/frappe/frappe/database.py", line 115, in connect
    charset='utf8mb4', use_unicode = True, conv = conversions, local_infile = self.local_infile)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/umoms/moms-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 630, in connect
    raise exc
OperationalError: (2003, "Can't connect to MySQL server on u'localhost' ([Errno 111] Connection refused)")

